README.md

```diff
@@ -1,10 +1,75 @@
 # mender-stress-test-client
```

Comment:
Once we are on it, can we add the GitLab build badge? (Copy it from any other repo)